SourceEditorCompose/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.microsoft.device.display.samples.sourceeditorcompose">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application
+        android:allowBackup="true"
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.TwoPaneExample"
+        tools:targetApi="31">
+        <activity
+            android:name="com.microsoft.device.display.samples.sourceeditorcompose.MainActivity"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:windowSoftInputMode="adjustPan"
+            android:theme="@style/Theme.TwoPaneExample">
```

Comment:
Same here, maybe you can use this? --> `android:theme="@style/Theme.ComposeSamples.NoActionBar"`

Reply:
I think what Cesar meant here is to use `@style/Theme.ComposeSamples.NoActionBar` which is different from `TwoPaneExample`.

Reply:
Please refer to other samples in this repo for the details.

Reply:
I saw how it was named in few other apps and refactored it to what Cesar suggested!